src/components/date/navbar/navbar.spec.js

```diff
@@ -1,30 +1,33 @@
 import React from 'react';
-import Navbar from './navbar';
+import TestRenderer from 'react-test-renderer';
 import { shallow } from 'enzyme';
+import 'jest-styled-components';
+import Navbar from './navbar.component';
+import ButtonStyle from './button.style';
```

Comment:
You should name this `StyledButton` as it' more intuitive and is consistent with its use in other files.

Reply:
Thanks, changed.

Reply:
I've changed it. Thanks